src/pages/Overview/OverviewCardSparkline.tsx

```diff
@@ -5,35 +5,46 @@ import { Metric } from '../../types/Metrics';
 import { getName } from '../../utils/RateIntervals';
 import { PFColors } from 'components/Pf/PfColors';
 import { SparklineChart } from 'components/Charts/SparklineChart';
-import { toVCLines } from 'utils/VictoryChartsUtils';
+import { toVCLine } from 'utils/VictoryChartsUtils';
 
 import 'components/Charts/Charts.css';
+import { RichDataPoint, VCLine } from 'types/VictoryChartInfo';
 
 type Props = {
   metrics?: Metric[];
+  errorMetrics?: Metric[];
   duration: DurationInSeconds;
 };
 
 class OverviewCardSparkline extends React.Component<Props, {}> {
   render() {
     if (this.props.metrics && this.props.metrics.length > 0) {
-      const data = toVCLines(this.props.metrics, 'rps', [PFColors.Blue400], 'time');
+      let series: VCLine<RichDataPoint>[] = [];
+
+      const data = toVCLine(this.props.metrics[0].datapoints, 'Total', PFColors.Blue400);
+      series.push(data);
+
+      if (this.props.errorMetrics && this.props.errorMetrics.length > 0) {
+        const dataErrors = toVCLine(this.props.errorMetrics[0].datapoints, '4XX+5XX', PFColors.Danger);
```

Comment:
I'd suggest to '4XX+5XX' -> '4xx+5xx'.
I think typically uses lowercase for the "x"
![image](https://user-images.githubusercontent.com/1662329/133768416-1937a9ba-a0c5-4f59-a8ff-cd1c703903d6.png)